changelog/bugfixes/2021-12-16-policycoreutils-fix-semodule-postinst.md

```diff
@@ -0,0 +1,10 @@
+The policycoreutils ebuild calls semodule in postinst to update SELinux stores.
```

Comment:
This changelog entry format is not really useful for the release notes. This paragraph is good for the PR description or a commit message, but here should be a bullet point for the release notes, e.g., `- SDK: Fixed build error popping up in the new SDK Container because `policycoreutils` used the wrong ROOT to update the SELinux store ([PR#1502](https://github.com/flatcar-linux/coreos-overlay/pull/1502))`
I'll file a PR for that here: https://github.com/flatcar-linux/coreos-overlay/pull/1537